Controller/CrudController.php

```diff
@@ -90,16 +88,7 @@ public function editAction(Request $request, $entity, $slug, $id)
         $relations = $this->get('opifer.crud.entity_helper')->getRelations($entity);
 
         // Set original relations, to be used after form's isValid method passed
-        foreach ($relations as $key => $relation) {
-            if ($relation['isOwningSide'] === false) {
-                $originalRelations[$key] = new ArrayCollection();
-                $getRelations = 'get' . ucfirst($relation['fieldName']);
-
-                foreach ($entity->$getRelations() as $relationEntity) {
-                    $originalRelations[$key]->add($relationEntity);
-                }
-            }
-        }
+        $originalRelations = $this->container->get('opifer.crud.relation_manager')->originalRelations([], $relations, $entity);
```

Comment:
In a controller you can simply do `$this->get('opifer.crud.relation_manager')->...`